cli/doc/node.rs

```diff
@@ -67,6 +68,14 @@ pub struct ModuleDoc {
   pub reexports: Vec<Reexport>,
 }
 
+#[derive(Debug, Serialize, Clone)]
+#[serde(rename_all = "camelCase")]
+pub struct ImportDef {
+  pub src: String,
+  pub local: String,
```

Comment:
Shouldn't this be in `DocNode::name` rather than an it being empty string? We could remove `InterfaceDef::local` then.